app/uk/gov/hmrc/exports/models/emails/SendEmailError.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models.emails
+
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.{HttpResponse, UpstreamErrorResponse}
+
+case class SendEmailError(status: Int, message: String)
```

Comment:
I am not I understand what can be gained by introducing this class? `EmailConnector` could return `HttpResponse` and this way provide some information about successful case to the client as well.

Reply:
Sorry, do not agree. Returning HttpResponse would move the logic to verify the response, which could be a success or a failure, to the client. What's wrong with doing that within the connector?

Reply:
I agree, we should not leak HttpResponse outside of a connector unless there is a compelling reason to.